oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/IndexerSupport.java

```diff
@@ -76,12 +76,21 @@ public class IndexerSupport {
     private File indexDefinitions;
     private final String checkpoint;
     private File existingDataDumpDir;
+    private long minModified;
```

Comment:
Should we use lastModifed  instead of minModified as we are using _modified property of mongMK which denotes lastModified timestamp.

Reply:
"Last modified" refers to the last time a node was modified.

"Min modified" means the minimum value of the "_modified" property.

Reply:
yes, _modified is independent of jcr:lastModified. My point was to just add info about what this property maps to.

Reply:
Yes, I understand now! Thanks!